components/p3a/nitro_utils/attestation.cc

```diff
@@ -35,16 +36,28 @@ namespace nitro_utils {
 
 namespace {
 
-constexpr char kHashPrefix[] = "sha256:";
-constexpr size_t kHashPrefixLength = 7;
-constexpr size_t kSHA256HashLength = 32;
-constexpr size_t kUserDataMinLength = kSHA256HashLength + kHashPrefixLength;
 constexpr size_t kAttestationBodyMaxSize = 16384;
+// The user_data field contains a multihash of the TLS cert
```

Comment:
We could link to https://multiformats.io/multihash/#the-multihash-format here for people unfamiliar with that format.

Reply:
Good idea, thanks! Added in 9b1cfccaca47fded6486b47f3fb306764d9e284c.